docs/index.js

```diff
@@ -50,20 +69,33 @@ function getDecodedName(name, code) {
   return decrypt(encryptedName, code);
 }
 
+function getGift(name, code) {
+  let gift = names.find(person => person.gift.toLowerCase() === gift.toLowerCase()).gift;
+  return decrypt(gift, code);
+}
+
 function getNameOrError(name, code) {
   //check to see if name is in names list
   let nameInList = names.find(person => person.name.toLowerCase() === name.toLowerCase());
   if (nameInList) {
     let decodedName = getDecodedName(name, code);
+    let personalGift = gifts[decodedName];
     //trim spaces from decoded name
     decodedName = decodedName.trim();
     //check to see if decoded name is in names list
     let decodedNameInList = names.find(person => person.name.toLowerCase() === decodedName.toLowerCase());
-    if (decodedNameInList) {
 
-      return `You are giving a gift to <br> <b style="font-size: 72pt">${decodedName}</b>`;
+      if (gifts[decodedName] === undefined) {
```

Comment:
instead of doing `gifts[decodedName]` here again, use your `personalGift` variable that you initialized earlier.